src/components/v2/appDetails/SyncError.component.tsx

```diff
@@ -57,8 +57,9 @@ const SyncErrorComponent: React.FC<SyncErrorType> = ({ appStreamData, showApplic
                 </span>
                 {collapsed && (
                     <span className="cn-9 ml-24 w-80 dc__ellipsis-right">
-                        {isImagePullBackOff && !appDetails.externalCi && 'IMAGEPULLBACKOFF'}
-                        {conditions.length > 0 && ', '}
+                        {isImagePullBackOff &&
+                            !appDetails.externalCi &&
+                            `IMAGEPULLBACKOFF${conditions.length > 0 && ', '}`}
```

Comment:
Is it working fine?

Reply:
Yes. Additional comma (,) was being displayed at the beginning when condition is not `IMAGEPULLBACKOFF`.